src/commands/plugins.rs

```diff
@@ -176,6 +187,10 @@ pub struct Upgrade {
     #[clap(short = 'y', long = "yes", takes_value = false)]
     pub yes_to_all: bool,
 
+    /// Overrides a failed compatibility check of the plugin with the current version of Spin.
+    #[clap(long = "override-compatibility-check", takes_value = false)]
```

Comment:
In the Install options you were using a const for this.  (I don't have strong feelings either way, just suggesting consistency, unless of course there is a reason not to.)